VirtoCommerce.Storefront/Startup.cs

```diff
@@ -369,18 +376,24 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 app.UseExceptionHandler("/error/500");
                 app.UseHsts();
             }
-            //Do not write telemetry to debug output 
+            // Do not write telemetry to debug output 
             TelemetryDebugWriter.IsTracingDisabled = true;
 
             app.UseResponseCaching();
 
             app.UseResponseCompression();
 
-            app.UseStaticFiles();
+            app.UseStaticFiles(new StaticFileOptions
```

Comment:
Could you explain why do you need it?

Reply:
it was an experiment with cache. i removed it